exercises/practice/isbn-verifier/.meta/example.sql

```diff
@@ -0,0 +1,31 @@
+UPDATE "isbn-verifier"
+   SET result = FALSE
+ WHERE GLOB('[0-9]-[0-9][0-9][0-9]-[0-9][0-9][0-9][0-9][0-9]-[0-9X]', isbn) = 0
```

Comment:
Aside: most the solutions I've seen don't care where the dashes are; they usually just remove all dashes.